lib/model/user.dart

```diff
@@ -0,0 +1,30 @@
+class User {
+  String _username;
+  String _password;
+  String token;
+  String message;
+  String userid;
+  User(this._username, this._password);
+
+  User.map(dynamic obj) {
```

Comment:
why is map used. i dont know

Reply:
map is used to assign json data to User object to make it access easily like user.userid.